src/Storage/Masa.Contrib.Storage.ObjectStorage.Aliyun/Client.cs

```diff
@@ -30,7 +30,7 @@ public Client(ICredentialProvider credentialProvider, IOptionsMonitor<AliyunStor
     /// <returns></returns>
     public TemporaryCredentialsResponse GetSecurityToken()
     {
-        if (!CredentialProvider.SupportSts)
+        if (CredentialProvider.IncompleteStsOptions)
             throw new ArgumentException($"Sts is not supported, {nameof(AliyunStsOptions.RegionId)} or {nameof(Options.RoleArn)} or {nameof(Options.RoleSessionName)} cannot be empty or null");
```

Comment:
Sts is not supported -> Sts options is imcomplete

Reply:
processed